source/connectors/kafka-installation.txt

```diff
@@ -0,0 +1,35 @@
+.. _kafka-installation:
+
+===============================
+Install MongoDB Kafka Connector
+===============================
+
+.. default-domain:: mongodb
+
+Install with Confluent Hub Client (Recommended)
+-----------------------------------------------
+
+1. Install the `Confluent Hub Client <https://docs.confluent.io/current/connect/managing/confluent-hub/client.html>`_
+   if necessary.
+
+2. Install the `MongoDB Connector for Apache Kafka <https://www.confluent.io/hub/mongodb/kafka-connect-mongodb>`_
```

Comment:
Is there anything else that goes into installing through Confluent Hub? Do I need to provide any sort of connection string or other config info?

Reply:
As far as installation, there is nothing further than what is described on the confluent page linked. The configuration, which I would consider separate from installation, depends on what they want to use it for and is described in detail in "Create a..." sections.

I would like to avoid copying over information on the external site if possible since it is subject to change, and is not a Confluent Hub Client is not a tool we manage. Would it be helpful if I reworded the sentence:

Old: "Install the MongoDB Connector for Apache Kafka component using the Confluent Hub Client."

New: "Follow the instructions on the MongoDB Connector for Apache Kafka Confluent page to install using the Confluent Hub Client".
?

Reply:
Sounds good. I like that update!